### PR TITLE
fix: make sure the links in the header stay white

### PR DESCRIPTION
**Pull request by hegeaal**
Fix because the links in the header was suddenly dark/not showing 🤔

### CODE REVIEW

File: src/components/header/styled.ts [hunk 1]
```diff
@@ -69,7 +69,7 @@ const NavigationLinks = styled.ul`
   margin-left: auto;
 
   & > li * {
-    color: ${theme.colour(Colour.NEUTRAL, 'N0')};
+    color: ${theme.colour(Colour.NEUTRAL, 'N0')} !important;
```
Comment (pooriamehregan) [inline src/components/header/styled.ts:72]: Burde helst unngå `!important`. Tror dette krever fiks i komponenten som tar prioritet ove denne komponenten.

**Review by pooriamehregan — CHANGES_REQUESTED** [review]

File: src/components/header/styled.ts (hunk 1 above)
Comment (jeffreiffers) [reply src/components/header/styled.ts:72]: https://github.com/fellesdatakatalog/fdk-kit/issues/200

**Review by jeffreiffers — APPROVED** [review]

**Comment by hegeaal** [comment]
@jeffreiffers @pooriamehregan Skal jeg bare lukke denne PR'en da?

**Comment by pooriamehregan** [comment]
@hegeaal Hm, jeg tror du kan lukke det ja.

**Pull request closed without merging**